spec/models/form/sales/pages/shared_ownership_type_spec.rb

```diff
@@ -1,13 +1,30 @@
-require "rails_helper"
+require "rails_hel\per"
 
 RSpec.describe Form::Sales::Pages::SharedOwnershipType, type: :model do
   subject(:page) { described_class.new(page_id, page_definition, subsection) }
 
   let(:page_id) { nil }
   let(:page_definition) { nil }
+  let(:subsection) { instance_double(Form::Subsection, form: instance_double(Form, start_date:)) }
   let(:start_date) { Time.utc(2022, 4, 1) }
-  let(:form) { instance_double(Form, start_date:) }
-  let(:subsection) { instance_double(Form::Subsection, form:) }
+
+  describe "headers" do
+    context "when 2023" do
```

Comment:
Can these two contexts have better descriptions please? e.g. "when the form year is 2023"

(My rule of thumb here is essentially read it as a sentence - I think you have done that elsewhere and maybe just missed these ones)